docs/core/install/linux-ubuntu.md

```diff
@@ -1,200 +1,181 @@
+| [Package manager<br>(Microsoft feed)*](#register-the-microsoft-package-repository) | <ul><li>Supported versions always available.</li><li>Preview releases are available.</li><li>Patches are available right way.</li><li>Dependencies are included.</li><li>Easy removal.</li></ul> | <ul><li>Requires registering the Microsoft package repository.</li><li>Only supports x64 Ubuntu.</li></ul> |
+| [Package manager<br>(Ubuntu feed)](#supported-distributions) | <ul><li>Usually the latest version is available.</li><li>Patches are available right way.</li><li>Dependencies are included.</li><li>Easy removal.</li></ul> | <ul><li>.NET versions available vary by Ubuntu version.</li><li>Preview releases aren't available.</li><li>Only supports x64 Ubuntu.</li></ul> |
+| [Snap](linux-snap.md) | <ul><li>Dependencies are included.</li><li>All supported CPU architectures available.</li><li>Easy to update to the latest patch version.</li><li>Easy removal.</li></ul> | <ul><li>Cumbersome to install and enable multiple runtimes.</li></ul> |
+| [Script \ Manual extraction](linux-scripted-manual.md) | <ul><li>Control where .NET is installed.</li></ul> | <ul><li>Manually install updates.</li><li>Manually install dependencies.</li><li>Manual removal.</li></ul> |
 
-## Supported distributions
+\* This is the recommended installation method for end-users and developers.
 
-The following table is a list of currently supported .NET releases and the versions of Ubuntu they're supported on.
+## Decide how to install .NET
 
-| Ubuntu                 | .NET       |
-|------------------------|------------|
-| [22.10](#2210)         | 7, 6       |
-| [22.04 (LTS)](#2204)   | 7, 6       |
-| [20.04 (LTS)](#2004)   | 7, 6       |
-| [18.04 (LTS)](#1804)   | 7, 6       |
-| [16.04 (LTS)](#1604)   | 6          |
+I want to install .NET because...
 
-[!INCLUDE [versions-not-supported](includes/versions-not-supported.md)]
+- **I want to create a .NET app:**
 
-## Install preview versions
+  We recommend you use APT and the Microsoft package repository. For more information, see the [Register and install with the Microsoft package repository](#register-the-microsoft-package-repository) section.
 
-[!INCLUDE [preview installs don't support package managers](./includes/linux-install-previews.md)]
+- **I want to run a .NET app in a container, cloud, or continuous-integration scenario:**
 
-## Remove preview versions
+  If your Ubuntu version provides the .NET version you require, install it from the built-in feed. Otherwise, [register the Microsoft package repository](#register-the-microsoft-package-repository) and install .NET from that repository. Review the information in the [supported distributions](#supported-distributions) section.
 
-[!INCLUDE [package-manager uninstall notice](./includes/linux-uninstall-preview-info.md)]
+  If the version of .NET you want isn't available, try using [dotnet-install script](linux-scripted-manual.md#scripted-install) or a [Snap](linux-snap.md) package.
```

Comment:
The Snap-based installation is very unhappy experience. There are number of active issues about it not working that we have been [closing as won't fix](https://github.com/dotnet/runtime/issues/51322#issuecomment-894449076). Would it be better to delete the Snap option from the documentation? Or at least say make its status clear.

Reply:
AFAIK we're not deemphasizing the Snap experience, despite the current issues. What would be the messaging to use as a warning?

Reply:
My preference would be to delete the Snap experience from the docs.  As far as I know, it is completely broken. I do not think that anybody can do anything useful with it. We can add it back once we find the time to fix the bugs. For example, this is what you get when try to install SDK according to the Snap docs and create hello world app on a fresh Ubuntu 22.04 LTS VM:

![image](https://user-images.githubusercontent.com/6668460/222618110-52eb3f63-87b6-4e5e-9073-3176b2ce8167.png)

Reply:
The snap experience out to be de-emphasized in docs and other guides. It leads users to failure and frustration. For example, see https://github.com/dotnet/core/issues/8254

Reply:
Removed crosslinks to snap doc.